Dockerfile.gnoland.release

```diff
@@ -1,13 +1,16 @@
 FROM busybox
 
-COPY ./gnoland /gnoland
+COPY ./gnoland /usr/bin/gnoland
 COPY ./examples /gnoroot/examples/
 COPY ./gnovm/stdlibs /gnoroot/gnovm/stdlibs/
 COPY ./gno.land/genesis/genesis_balances.txt /gnoroot/gno.land/genesis/genesis_balances.txt
 COPY ./gno.land/genesis/genesis_txs.jsonl /gnoroot/gno.land/genesis/genesis_txs.jsonl
 
 ENV GNOROOT="/gnoroot/"
 
-EXPOSE 26657 36657
-ENTRYPOINT [ "/gnoland" ]
-CMD [ "" ]
\ No newline at end of file
+WORKDIR /gnoroot/
+
+EXPOSE 26657 26657
```

Comment:
Does this mean we standardized `26657` as the default for our Docker images?

Reply:
```suggestion
EXPOSE 26657
```

Reply:
https://github.com/gnolang/gno/pull/2178 I believe this say so :)